packages/web-components/src/accordion/accordion.ts

```diff
@@ -181,69 +194,7 @@ export class Accordion extends FASTElement {
         this.setSingleExpandMode(item);
       }
 
-      this.change();
-    }
-  };
-
-  private getItemIds(): Array<string | null> {
-    return this.slottedAccordionItems.map((accordionItem: HTMLElement) => {
-      return accordionItem.id;
-    });
-  }
-
-  private isSingleExpandMode(): boolean {
-    return this.expandmode === AccordionExpandMode.single;
-  }
-
-  private handleItemKeyDown = (event: KeyboardEvent): void => {
```

Comment:
Are we going to run into trouble given https://www.w3.org/WAI/ARIA/apg/patterns/accordion/?

Reply:
I spoke with @smhigley and the tl;dr is keyboard behavior should match expectations for the semantics of a control and arrowing doesn't match the semantics used for Accordion.

Removing focusgroup also puts us on par with the native `details[name]` behavior.

I tend to think less-is-more when it comes to a11y but if there's historical reasons to abide strictly by the APG, I'm happy to revert these changes.

Reply:
If we have parity with the react version and alignment with @smhigley, then I think we're clear and have done our due diligence :)